pkg/sink/kafka/admin.go

```diff
@@ -58,15 +58,15 @@ func (a *saramaAdminClient) GetAllBrokers() ([]Broker, error) {
 	return result, nil
 }
 
-func (a *saramaAdminClient) GetCoordinator() (int32, error) {
+func (a *saramaAdminClient) GetCoordinator(context.Context) (int, error) {
```

Comment:
why change int32 to int?

Reply:
because it's more easy to handle in the kafka-go implementation.